Shared/sdk/SharedUtil.Hash.h

```diff
@@ -30,7 +30,9 @@ enum class PasswordHashFunction
 
 enum class StringEncryptFunction
 {
-    TEA
+    TEA,
+    BASE32,
+    BASE64
 };
```

Comment:
The `StringEncryptFunction` name is a bit odd due to `base64` not being an encryption method. Maybe we should change this to `StringEncodeFunction`?

Reply:
Thanks for review. I'll edit.

Reply:
@cleopatradev This change is still pending.

Reply:
@cleopatradev This is still an issue.